.github/workflows/coq-action.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    # the OS must be GNU/Linux to be able to use the docker-coq-action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - mathcomp/mathcomp:1.10.0-coq-8.10
```

Comment:
That I meant today is that you could keep the non-dev images from 8.7, but drop the corresponding -dev one (since it is going to break soon).

Reply:
Ah OK, thanks @gares! I'll force-push a new version ASAP